.github/workflows/delete_pr_documentation.yml

```diff
@@ -0,0 +1,72 @@
+name: Delete PR Documentation
+
+on:
+  workflow_call:
+    inputs:
+      package_name:
+        required: true
+        type: string
+    secrets:
+      hf_token:
+        required: true
+
+jobs:
+  upload_pr_documentation:
+    runs-on: ubuntu-latest
+    if: >
+      github.event.workflow_run.event == 'pull_request' &&
+      github.event.workflow_run.conclusion == 'success'
+
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          repository: 'huggingface/doc-builder'
+          path: doc-builder
+          ref: fix-token-issue
+
+      - name: Setup environment
+        shell: bash
+        run: |
+          pip install black
+          pip uninstall -y doc-builder
+          cd doc-builder
+          git pull origin fix-token-issue
+          pip install .
+          cd ..
+          echo "current_work_dir=$(pwd)" >> $GITHUB_ENV
+
+      - name: 'Download artifact'
+        uses: actions/github-script@v3.1.0
+        with:
+          script: |
+            var artifacts = await github.actions.listWorkflowRunArtifacts({
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+              run_id: ${{github.event.workflow_run.id }},
+            });
```

Comment:
Is it possible to get the PR number in a similar fashion?

(Currently someone can modify the workflow and overwrite the docs of another PR)

Reply:
For example, `${{github.event.workflow_run.pull_requests[0].number}}`

Reply:
looking into it

Reply:
based on this discussion https://github.com/orgs/community/discussions/25220, it doesn't seem so
> Is it possible to get the PR number in a similar fashion?

Reply:
Did you try `${{github.event.workflow_run.pull_requests[0].number}}`?

https://docs.github.com/en/webhooks-and-events/webhooks/webhook-events-and-payloads#workflow_run

![image](https://github.com/huggingface/doc-builder/assets/342922/d0c0455c-13a8-410d-9705-2f74b5af836b)

It may have been added after the discussion

Reply:
https://github.com/huggingface/accelerate-wip/actions/runs/5188864703 testing [d955206](https://github.com/huggingface/doc-builder/pull/379/commits/d955206cfa72a64567cd4e02ff23fe04f84a26ee)

Reply:
![Uploading image.png…]()